app/user.go

```diff
@@ -2236,3 +2222,21 @@ func (a *App) getListOfAllowedChannelsForTeam(teamId string, viewRestrictions *m
 
 	return listOfAllowedChannels, nil
 }
+
+// invalidateUserCacheAndPublish Invalidates cache for a user and publishes user updated event
+func (a *App) invalidateUserCacheAndPublish(userId string) {
+	a.InvalidateCacheForUser(userId)
+
+	user, userErr := a.GetUser(userId)
+	if userErr != nil {
+		mlog.Error(fmt.Sprintf("Error in getting users profile for id=%v", userId), mlog.String("user_id", userId))
```

Comment:
You might want to log `userErr` here. 0/5

Reply:
whoops, thought it was, good catch. Copy paste :)